projects/fsastorefrontlib/src/core/checkout/facade/checkout.service.ts

```diff
@@ -37,24 +37,18 @@ export class FSCheckoutService extends CheckoutService {
   orderPlaced: boolean;
   mockedDeliveryMode = 'financial-default';
 
-  setIdentificationType(identificationType: string) {
-    combineLatest([
-      this.activeCartService.getActiveCartId(),
-      this.userIdService.getUserId(),
-    ])
-      .subscribe(([activeCartCode, occUserId]) => {
```

Comment:
Subscription in services moved to component. Service is not only responsible for dispatching an action.